src/components/form-helpers/form-helpers.js

```diff
@@ -321,17 +324,10 @@ const _PhoneFieldItem = ({
       autoPlaceholder: 'aggressive',
       preferredCountries: ['ar', 'mx', 'co', 'es', 'ec', 'cl', 'pe', 'us'],
       initialCountry: 'auto',
-      geoIpLookup: async (success, failure) => {
-        // TODO: read cache before request
-        try {
-          const response = await axios.get('https://ipinfo.io/json');
-          var countryCode = response.data && response.data.country ? response.data.country : 'AR';
-          success(countryCode);
-          console.log(response);
-        } catch (error) {
-          success('ES'); // TODO
-          console.log(error.toJSON());
-        }
+      geoIpLookup: async (success) => {
+        const response = await ipinfoClient.getCountryCode();
+        const countryCode = response.data && response.data.country ? response.data.country : 'AR';
```

Comment:
Could you move the request details inside the service? For example:

```suggestion
        const codeOrNull = await ipinfoClient.getCountryCode();
        const countryCode = codeOrNull || 'AR';
```

or better yet:

```suggestion
        const countryCode = await ipinfoClient.getCountryCode({ fallback: 'AR'});
```